lib/Runtime/Library/WebAssemblyModule.cpp

```diff
@@ -154,9 +154,7 @@ WebAssemblyModule::EntryImports(RecyclableObject* function, CallInfo callInfo, .
     module->m_imports->Map([AddImport] (int index, Wasm::WasmImport * import) {
         AddImport(import, Wasm::ExternalKinds::Function);
     });
-    module->globals->Map([AddImport](int index, Wasm::WasmGlobal * global) {
-        AddImport(global->importVar, Wasm::ExternalKinds::Global);
-    });
+    // Todo:: Add globals (old implementation didn't work)
```

Comment:
I already have a change pending that fixes this. I will open a PR after this is merged